models/userDao.js

```diff
@@ -0,0 +1,25 @@
+const { myDataSource } = require("./myDataSource");
+
+const createUser = async (email, hashedPassword, name, mobile) => {
+  try {
+    return await myDataSource.query(
+      `INSERT INTO users(
+		    email,
+		    password,
+        name,
+		    mobile
+		) VALUES (?, ?, ?, ?);
+		`,
+      [email, hashedPassword, name, mobile]
+    );
+  } catch (err) {
+    console.log(err);
+    const error = new Error("INVALID_DATA_INPUT");
+    error.statusCode = 500;
```

Comment:
status code 500은 에러처리를 못 한 경우에 발생하게 하는게 적절합니다.
따라서 에러를 핸들링 했을 경우 4xx code를 사용하면 좋을 것 같습니다.
```suggestion
    error.statusCode = 400;
```

Reply:
수정했습니다!